### ISSUE TITLE
cannot set og:title

### ISSUE THREAD

**Issue by adrelanos** [octfx/wiki-seo#28]
I cannot set `og:title`. It is always set to the name of the wiki page.

For example wiki page `wiki/NameOfWikiPage` will result in:

> <meta property=og:title content=NameOfWikiPage />

```
{{#seo:
|title=Test-Title2
|description=Description is functional.
}}
```

It properly sets the HTML title though:

> `<title>Test-Title2</title>`

The following does not help either:

```
{{#seo:
|title=Test-Title2
|og:title=Test-Title2
|description=Description is functional.
}}
```

Using `{{DISPLAYTITLE:Test-Title2}}` above or below the `{{#seo:` code block ​results in the bold title on top of the wiki page to show `Test-Title2` as expected but `og:title` is ​still wrong, i.e. still `<meta property=og:title content=NameOfWikiPage />`.

----

Using:

* mediawiki 1.35.2
* WikSEO 2.6.5 (202fdb8)

```
$wgAllowDisplayTitle = true;
$wgRestrictDisplayTitle = false;
```

**Comment by octfx** [comment]
I think this is due to the following PR #19 
I've pushed a fix to gerrit, I'll back merge this when all tests are run

**Comment by octfx** [comment]
Should be fixed in latest commit

**Comment by adrelanos** [comment]
Thank you!

Works for me!

Could you please merge into / backport to the 1_36 versioned branch?

**Comment by octfx** [comment]
Sure! 
Just merged to 1.36
